examples/Components/forcefield/TetrahedralCorotationalFEMForceField.scn

```diff
@@ -1,39 +1,74 @@
 <?xml version="1.0" ?>
 <!-- Mechanical Tetrahedral Corotational FEM Basic Example -->
-<Node name="root" dt="0.05" showBoundingTree="0" gravity="0 0 0">
-    <RequiredPlugin pluginName="SofaOpenglVisual"/>
+<Node name="root" dt="0.05" showBoundingTree="0" gravity="0 -9 0">
     <RequiredPlugin pluginName='SofaBoundaryCondition'/>
+    <RequiredPlugin pluginName='SofaEngine'/>
```

Comment:
- `name` instead of `pluginName`
- double quotes instead of single quotes

This should apply in all your changes